cmd/awscollector/Dockerfile

```diff
@@ -26,18 +28,7 @@ FROM scratch
 
 COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=build /workspace/build/linux/aoc_linux /awscollector
-COPY config.yaml /etc/otel-config.yaml
```

Comment:
Does this map 1 to 1? Let us hope we don't break anything.

Reply:
It does. I did a comparison with dive.
<img width="1900" alt="image" src="https://user-images.githubusercontent.com/84729962/149425507-0b9d4c1a-8d32-46e5-8f74-b2f3476dfd3c.png">